WordPress/WordPressScreenshotGeneration/WordPressScreenshotGeneration.swift

```diff
@@ -8,6 +9,7 @@ class WordPressScreenshotGeneration: XCTestCase {
         super.setUp()
 
         // Put setup code here. This method is called before the invocation of each test method in the class.
+        SDStatusBarManager.sharedInstance()?.enableOverrides()
```

Comment:
Why is this needed in addition to `IS_GENERATING_SCREENSHOTS=1` in the `Snapfile`?